applications/nrf5340_audio/broadcast_sink/main.c

```diff
@@ -451,9 +452,14 @@ int main(void)
 
 	LOG_DBG("nRF5340 APP core started");
 
+	channel_assignment_init();
```

Comment:
Could we bring this below `nrf5340_audio_dk_init()`?

Reply:
Unfortunately not, `nrf5340_audio_dk_init()` calls `channel_assignment_set()`, so I assume the channel assignment module should be initialized before that call

Reply:
What about moving the `channel_assignment_init` into `nrf5340_audio_common_init` and then move that above the `nrf5340_audio_dk_init`? I don't see anything right off the bat that should prevent that at least.

Reply:
This worked out pretty well! The only blocker I hit was calling `nrf5340_audio_common_init` before `nrf5340_audio_dk_init`, as the `bt_mgmt_init()` call in audio common requires the button handler to be initialized. So I kept the order, but moved `channel_assignment_init()` and `audio_system_init()` into audio common.